web/impact/impact/urls.py

```diff
@@ -81,6 +82,8 @@
     url(r'^oauth/',
         include('oauth2_provider.urls', namespace='oauth2_provider')),
     url(r'^schema/$', schema_view, name='schema'),
+    url(r'^directory/$', TemplateView.as_view(template_name='directory.html'),
```

Comment:
I made this a dumb static view for now but I should likely change this to an authenticated view. I've not written python or Django before so went with the simplest solution to get started.

Reply:
we should probably make the mentor directory a sub-url of directory unless we plan on serving the startup and mentor directory from a single directory.html template.

Reply:
We have a couple of options, we could redirect any path that starts with `/directory` and let react take over from there or we could use a subdomain. I'm not familiar enough with django but I'd presume `/directory*` will be simplier then a subdomain.

We can serve a single index page that boots the react app and that's fine.

Reply:
Auth would be easy to add here: wrap the `TemplateView` with the `login_required` function -- [Django docs have an example](https://docs.djangoproject.com/en/1.10/topics/class-based-views/intro/#decorating-in-urlconf).

Seems like we need a design decision on the URL structure though.